lib/mocha/stubbed_method.rb

```diff
@@ -8,9 +8,9 @@ class StubbedMethod
 
     def initialize(stubbee, method_name)
       @stubbee = stubbee
+      @method_name = method_name
       @original_method = nil
       @original_visibility = nil
-      @method_name = method_name.to_sym
```

Comment:
How do you know it's safe to make this change and that it won't result in a change in behaviour under some circumstances? Why is it safe to change this call, but not other calls to `String#to_sym` in the rest of the codebase?

Reply:
I was relying on existing tests to catch issues, but now I think it's better to leave this conversion in place here. For the most part, it should work either way since the `method_name` attr of `StubbedMethod` is ultimately gets passed to ruby library methods that'll accept either a symbol or a string (e.g. `define_method`, `remove_method`, etc.). However, `StubbedMethod#matches` might behave differently if one instance has the `method_name` as string and the other has it as a symbol.

Hence, I've undone this change.

Reply:
Thanks. I know I'm probably being excessively cautious and I know I should rely on the tests to make these kinds of changes, but I've been burned so many times by people relying on undocumented behaviour that I'd like to be more confident that a change like this isn't going to break something accidentally.